### ISSUE TITLE
Keyper Config.Unmarshal() seems broken

### ISSUE THREAD

**Issue by RmbRT** [shutter-network/shutter#127]
During my work on the TEE fortification of the keyper, I noticed there was no test for loading a TOML config. I wrote a test and it seems like somehow, the Viper object, while containing all the keys and values, is unable to properly transfer those into the Config object.

Calling .Unmarshal() on an empty Config object results in a segfault, because the keys are nil. If I GenerateKeys() and then call Unmarshal(), it does not panic, but all values seem to be left as 0's (default initialised), and the keys remain the same as when they were generated.

I wrote a test case for that in the TEE branch over at perun-network/shutter.

**Comment by faheelsattar** [comment]
Hey @RmbRT just went through what you reported and the tests aswell. It looks like in your test the viper instance isnt intialized correctly which might be causing the issue (https://github.com/perun-network/shutter/commit/a164f068d0a2b65470d813e526d942812fcfeab0#diff-a85a75093bd9992cbeb1ef0cae5791a03190c572f89cb3664b19e0fa5d03bf8eR47). Can you try replacing it with `v := viper.New()`. Let me know how that works out and we can move on from there with this issue :)

**Comment by faheelsattar** [comment]
Closing this issue as it doesn't appear to be broken, feel free to reopen if you still encounter it after updating the tests

**Comment by RmbRT** [comment]
Thanks. Typical Go moment… Thought of so many things but not about whether there was a dedicated .New() method. I'll fix it and report back later today.

**Comment by RmbRT** [comment]
Yes, this fixed it. Thank you. Pushed a new commit in the perun-network/shutter fork (`shutter-tee` branch).

**Comment by faheelsattar** [comment]
> Yes, this fixed it. Thank you. Pushed a new commit in the perun-network/shutter fork (`shutter-tee` branch).

Great!
